build_r.R

```diff
@@ -74,7 +104,7 @@ result <- file.copy(
 #       by CMake while also meeting the CRAN req to create object files
 #       on demand
 cmd <- "R CMD build lightgbm_r --keep-empty-dirs"
```

Comment:
`cmd` is not used anymore?

Reply:
ah you're right we can remove it now

Reply:
removed in https://github.com/microsoft/LightGBM/pull/2965/commits/828023f8c04002e00bfe66e43069bb383e7e0fea

Reply:
Ah whoops, we do actually use it! To print the command when you run with `--skip-install`. I'll put it back.

![image](https://user-images.githubusercontent.com/7608904/81514580-ccf77b00-9327-11ea-9b3c-ae96e236169a.png)

Reply:
or more accurately...we had two `cmd` in that part of the code and needed to delete one...but I deleted the wrong one! fixed in https://github.com/microsoft/LightGBM/pull/2965/commits/f9e1a5203966c9f8d24df38ebfd2eb054248c699